docs/04-System-Hardening/15-UFW-Firewall-Basics.md

```diff
@@ -10,6 +10,12 @@ In this section, we will take a look at UFW Firewall Basics
 
 ### UFW Rules
 
+    ufw status
```

Comment:
@M-Ayman  these should be in command/code format. Please change all commands to code format.